### DISCUSSION TITLE
table data search

### DISCUSSION THREAD

**Discussion by zfranco55** [themesberg/flowbite-svelte#1354] [General]
Sorry for my programming ignorance, but I wanted to ask how I can search on all columns (or at least more than one).
I only found this example :
` $: filteredItems = items.filter((item) => item.maker.toLowerCase().indexOf(searchTerm.toLowerCase()) !== -1);`

**Comment by sanjusoftware** [comment]
just extend the logic to other columns like below
`$: filteredItems = items.filter((item) => item.maker.toLowerCase().indexOf(searchTerm.toLowerCase()) !== -1 ||
item.col2.toLowerCase().indexOf(searchTerm.toLowerCase()) !== -1 ||
item.maker.col3.toLowerCase().indexOf(searchTerm.toLowerCase()) !== -1 );`
